static/app/views/replays/deadRageClick/deadRageSelectorCards.tsx

```diff
@@ -64,15 +56,16 @@ function RageClickTable({location}: {location: Location<any>}) {
     sort: '-count_rage_clicks',
     cursor: undefined,
     prefix: 'selector_',
+    isWidgetData: true,
   });
 
   return (
     <SelectorTable
-      data={data}
+      data={data.filter(d => (d.count_rage_clicks ?? 0) > 0)}
```

Comment:
Is there a long term plan to handle this on the backend?

Reply:
we should have one! cc @cmanallen is it possible to add the ability to query for top selectors such that `count_rage/dead_clicks > 0`?

Reply:
Yeah.  Should that be the default behavior or do you want to provide a query parameter which enables that filter?